config.toml

```
@@ -1,4 +1,4 @@
-baseURL = "https://nextgems.pages.gwdg.de/global-hackathon-hamburg/"
+baseURL = "https://nextgems.pages.gwdg.de/global-hackathon-template/"
```

This url wasn't up to date to begin with. Good that the deployment worked anyway. Maybe change to something completely generic?
```suggestion
baseURL = "https://digital-earths-global-hackathon.github.io/template-node/"
```